### ISSUE TITLE
Cannot execute neon build on Ubuntu or Raspberry Pi Linux envs

### ISSUE THREAD

**Issue by jugglingcats** [neon-bindings/neon#209]
I'm trying to get a Neon project running on RPi without success. I get the following error:
```
~$ mkdir tmp
~$ cd tmp
~/tmp$ neon new neon-test
...
~/tmp$ cd neon-test
~/tmp/neon-test$ neon build
neon ERR! Cannot read property 'length' of undefined
```

Neon version is 0.1.17. I've tried the same on Raspberry Pi (Raspbian OS) and vanilla Ubuntu 16.04.1 LTS. On Windows it's fine (same Neon version on all).

Running `cargo build` in the `native` directory is successful.

**Comment by jugglingcats** [comment]
Apologies, I think this should have been raised on the neon-cli project!

**Comment by jugglingcats** [comment]
Looks like node version issue... my Pi and Ubuntu both have node 7.x and when I run this on Windows I get the same error.

**Comment by jugglingcats** [comment]
Hmmm...

```
pi@alfie-pi-2:~/tmp/neon-test $ node --version
v6.6.0
pi@alfie-pi-2:~/tmp/neon-test $ neon build
neon ERR! Cannot read property 'length' of undefined
```

I switched node version using `nvm`. I wonder if neon is always taking node from `/usr/bin/node` (still version 7.x) rather than from my path.

**Comment by jugglingcats** [comment]
I downgraded the node apt package using the following commands:

```
$ curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -

$ apt-cache policy nodejs
nodejs:
  Installed: 7.5.0-1nodesource1~jessie1
  Candidate: 7.5.0-1nodesource1~jessie1
  Version table:
 *** 7.5.0-1nodesource1~jessie1 0
        100 /var/lib/dpkg/status
     6.10.3-1nodesource1~jessie1 0
        500 https://deb.nodesource.com/node_6.x/ jessie/main armhf Packages
     0.10.29~dfsg-2 0
        500 http://mirrordirector.raspbian.org/raspbian/ jessie/main armhf Packages

$ sudo apt-get install nodejs=6.10.3-1nodesource1~jessie1
```

Now I have:
```
pi@alfie-pi-2:~ $ whereis node
node: /usr/bin/node /usr/local/bin/node /usr/include/node /home/pi/.nvm/versions/node/v6.6.0/bin/node /usr/share/man/man1/node.1.gz
pi@alfie-pi-2:~ $ /usr/bin/node -v
v6.10.3
pi@alfie-pi-2:~ $ /usr/local/bin/node -v
v6.6.0
pi@alfie-pi-2:~ $ /home/pi/.nvm/versions/node/v6.6.0/bin/node -v
v6.6.0
```

All versions are 6.x but am still getting the same error for `neon build` :(

**Comment by passcod** [comment]
Looks like neon-cli 0.1.17 is broken. 0.1.15 works.

**Comment by dherman** [comment]
Yeah, my apologies. It's a CLI bug in 0.1.17. I just fixed it today and I'll spin up a new release ASAP.

**Comment by dherman** [comment]
OK, I published 0.1.18. Try updating and see if it's fixed for you?

**Comment by jugglingcats** [comment]
It's fixed for me - thanks!
